streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -1166,8 +1166,8 @@ public void cleanUp() {
      * The returned object can be used to query the {@link StateStore} instances.
      *
      * Only permits queries on active replicas of the store (no standbys or restoring replicas).
-     * See {@link KafkaStreams#store(java.lang.String, org.apache.kafka.streams.state.QueryableStoreType, boolean)}
-     * for the option to set {@code includeStaleStores} to true and trade off consistency in favor of availability.
+     * See {@link KafkaStreams#store(StoreQueryParams)}
+     * for the option to set {@code StoreQueryParams.withIncludeStaleStores()} and trade off consistency in favor of availability.
```

Comment:
As we deprecate this method, I would remove the whole paragraph (that was added via KIP-535 and not necessary any longer)

Reply:
Done.